lib/src/vector_math/vector2.dart

```diff
@@ -97,9 +102,9 @@ class Vector2 implements Vector {
   /// Check if two vectors are the same.
   @override
   bool operator ==(Object other) =>
-      (other is Vector2) &&
-      (_v2storage[0] == other._v2storage[0]) &&
-      (_v2storage[1] == other._v2storage[1]);
+      other is Vector2 &&
+      _v2storage[0] == other._v2storage[0] &&
+      _v2storage[1] == other._v2storage[1];
```

Comment:
flip order (accessing [1] before [0]

Reply:
Alright, I naively thought that the compiler would handle it when the index wasn't dynamic.

Reply:
I flipped the order of all of them now.